src/components/Accordion/Accordion.test.tsx

```diff
@@ -9,16 +10,22 @@ import * as stories from './Accordion.stories';
 const { Default } = composeStories(stories);
 
 describe('<Accordion />', () => {
-  generateSnapshots(stories);
+  generateSnapshots(stories as StoryFile);
```

Comment:
I notice some usages of `generateSnapshots` require `as StoryFile` and others don't, do you know what's happening here?

Reply:
not exactly sure, but it could be the type changes related to some of the storybook libraries. The first sweep was to satisfy typescript to restore parity on `yarn test`, and we can see if something systematic is causing certain story files to need a type